plasmapy/particles/species.py

```diff
@@ -0,0 +1,35 @@
+import astropy.units as u
+
+from dataclasses import dataclass
+from typing import Optional
+
+from plasmapy.formulary import gyrofrequency, mass_density, thermal_speed
+
+from . import particle_class
+
+
+@dataclass
```

Comment:
Python 3.7! 😺 😺 😺 😺 😺 

I think a dataclass is a good choice here, at the very least to start out with.